tools/hab-channel-list-pkgs/hab-channel-list-pkgs.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+#
+# Returns a list of all fully qualified package identifiers that exist in a
+# given release channel. Prints the package identifiers to standard out, one
+# entry per line.
+#
+set -eu
+if [ -n "${DEBUG:-}" ]; then set -x; fi
+
+main() {
+  need_cmd curl
+  need_cmd jq
+
+  # Set the default Builder URL
+  : ${HAB_BLDR_URL:=https://bldr.habitat.sh}
+  # Channel name is required
+  channel="${1:-}"
+  if [[ -z "$channel" ]]; then
+    echo "Usage: $(basename $0) <CHANNEL>" >&2
+    exit 1
+  fi
+  # Calculate the channel URL from which to fetch
+  url="$HAB_BLDR_URL/v1/depot/channels/core/$channel/pkgs"
```

Comment:
Might want to either document this tool as pertaining just to the `core` origin, or allow for an optional `origin` argument (defaulting, of course, to `"core"`).

Reply:
Oh right, too tired last night to have caught that. I'm going to push this with an update that'll keep it even more generally useful.